sandbox/tests/test scenes/procedural objects/09 - lit sphere - mesh.appleseed

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- File generated by appleseed.dll version 2.1.0-beta (SSE SSE2 SSE3 SSSE3 SSE4.1 SSE4.2). -->
+<project format_revision="31">
+    <scene>
+        <camera name="camera" model="pinhole_camera">
+            <parameter name="autofocus_enabled" value="false" />
+            <parameter name="controller_target" value="0 0 0.00217694" />
+            <parameter name="film_dimensions" value="0.024892 0.018669" />
+            <parameter name="focal_length" value="0.035" />
+            <transform time="0">
+                <matrix>
+                    0.53723399176755826 -0.52982861269911252 -0.65624787942882457 -4.65442925275551644
+                    -0.00000000000000000 0.77806736174274815 -0.62818085022601544 -4.45536422587390657
+                    0.84343324459585733 0.33748010571886095 0.41800423461310920 2.96686634829537343
+                    0.00000000000000000 0.00000000000000000 0.00000000000000000 1.00000000000000000
+                </matrix>
+            </transform>
+        </camera>
+        <environment name="environment" model="generic_environment" />
+        <assembly name="assembly">
+            <bsdf name="bsdf1" model="lambertian_brdf">
+                <parameter name="reflectance" value="0.5" />
+                <parameter name="reflectance_multiplier" value="1.0" />
+            </bsdf>
+            <material name="material1" model="generic_material">
+                <parameter name="bsdf" value="bsdf1" />
+                <parameter name="bump_amplitude" value="1.0" />
+                <parameter name="displacement_method" value="bump" />
+                <parameter name="normal_map_up" value="z" />
+            </material>
+            <light name="light1" model="directional_light">
+                <parameter name="cast_indirect_light" value="true" />
+                <parameter name="exposure" value="0.0" />
+                <parameter name="importance_multiplier" value="1.0" />
+                <parameter name="irradiance" value="1" />
+                <parameter name="irradiance_multiplier" value="2" />
+            </light>
+            <object name="sphere" model="mesh_object">
+                <parameter name="primitive" value="sphere" />
+                <parameter name="radius" value="1.0" />
+                <parameter name="resolution_u" value="100" />
+                <parameter name="resolution_v" value="100" />
+            </object>
+            <object_instance name="sphere_inst" object="sphere">
+                <transform>
+                    <translation value="0.0 0.0 -1.0" />
+                </transform>
+                <assign_material slot="default" side="front" material="material1" />
+            </object_instance>
+        </assembly>
+        <assembly_instance name="assembly_inst" assembly="assembly">
+                <transform>
+                    <rotation axis="1.0" angle="90.0" />
```

Comment:
I'm surprised that this works at all, `axis` is supposed to contain three values (X, Y, Z).

Reply:
Also, could you load and save again the file in appleseed.studio (or using `projecttool`) please?

Reply:
I did and now it's a marix